src/main/scala/ScalaModulePlugin.scala

```diff
@@ -147,6 +147,17 @@ object ScalaModulePlugin extends AutoPlugin {
 
   lazy val scalaModuleSettingsJVM: Seq[Setting[_]] = scalaModuleOsgiSettings
 
+  lazy val scalaModuleSettingsNative: Seq[Settings[_]] = Seq(
+    scalaVersion := "2.11.11",
+    skip in compile := System.getProperty("java.version").startsWith("1.6"),
+    test := {},
+    libraryDependencies := {
+      if (!scalaVersion.value.startsWith("2.11"))
+        libraryDependencies.value.filterNot(_.organization == "org.scala-native")
+      else libraryDependencies.value
+    }
```

Comment:
I defo agree on the fact that cross compilation in multi-module projects is broken once you pass the trivial setup. I did used in the past `sbt-doge` and solves the problem, but also did not think that adding it to the parser combinators module would be a good idea. Besides, the pure existance of a plugin like `sbt-doge` is a clear sign of _something lacking_ in SBT ecosystem.

Anyway, hopefully something that would be fixed (or less hacky) in SBT 1.0...